docs/reference/commandline/swarm_init.md

```diff
@@ -128,6 +129,11 @@ management traffic of the cluster.
 If unspecified, Docker will use the same IP address or interface that is used for the
 advertise address.
 
+### `--default-addr-pool`
+This flag specifies default subnet pools for global scope networks. If subnet size is not
+specified then default value 24 will be used.
+If unspecified, Docker will use the predefined subnets as it works on older releases.
```

Comment:
Probably worth repeating the format description here, but with more info.

Reply:
added sample example